.github/workflows/greetings.yml

```diff
@@ -8,6 +8,6 @@ jobs:
     steps:
     - uses: actions/first-interaction@v1
       with:
-        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        repo-token: ${{ secrets.PA_TOKEN }}
```

Comment:
@nellyk is this change to fix the bug on the first interaction security issue? did you test it?

Reply:
You are right, i added for it to be used for it to be tied to the @oneamexbot  account, i'll remove it

Reply:
removed. Thank you!